templates/projects/tasks_browse.html

```diff
@@ -33,6 +33,7 @@
               {% endif %}
               <li><strong>{{_('Actions')}}</strong>: {{_('Three current actions: download, view, or get user info for task results')}}.</li>
               <li><strong>{{_('Active Users')}}</strong>: {{_('List users working on the task')}}.</li>
+              <li><strong>{{_('Completed By')}}</strong>: {{_('List users that completed the task')}}.</li>
```

Comment:
This file is generated by .ejs file. Technically, we don't have to change this file if we changed the corresponding .ejs file. But there is no harm to change it also.

Reply:
Indeed. The ejs update is [included](https://github.com/bloomberg/pybossa-default-theme/pull/381/files#diff-d71135813b0ca1c5f762d349de655d16bb91bbbd5cc8728754ee542e046287a2) as well.